.github/workflows/pr-e2e-tests.yml

```
@@ -0,0 +1,356 @@
+          node-version: ${{ env.NODE_VERSION }}
+      - name: Build frontend for cypress
+        if: steps.cypress-build-cache.outputs.cache-hit != 'true'
+        run: npm run cypress:server:build:coverage
+        working-directory: ./frontend
+
+  Launch-Dashboard:
+    needs: [Setup, Cypress-Setup]
+    if: needs.check-trigger.outputs.should_run == 'true'
+    runs-on: self-hosted
+    outputs:
+      dashboard-url: ${{ steps.get-url.outputs.url }}
```

actionlint (1.7.7)

property "get-url" is not defined in object type {launch: {conclusion: string; outcome: string; outputs: {string => string}}}

(expression)